KsApi/models/Project.swift

```diff
@@ -50,6 +50,8 @@ public struct Project {
   public struct Stats {
     public let backersCount: Int
     public let commentsCount: Int?
+    public let currentCurrency: String?
+    public let currentCurrencyRate: Float?
```

Comment:
Our API now returns the "current" currency of the response (which should be pass-through from the request), and the exchange rate against the project currency.